src/main/resources/locales/cy/translation/contact-the-tribunal.json

```diff
@@ -1,104 +1,104 @@
 {
-  "title": "Welsh Translation required",
-  "h1": "Welsh Translation required",
+  "title": "Cysylltu â’r tribiwnlys ynglŷn â’ch achos",
```

Comment:
As you added translations for the contact the tribunal page, could you please add the feature flag to the controller?

Reply:
Added feature flag